### PR TITLE
IBX-7869: Content Type -> content type - capitalization deleted / Developer Documentation

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-7869)
| Versions      | master

"Content Type" - "content type"

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: code_samples/api/public_php_api/src/Command/CreateContentTypeCommand.php [hunk 1]
```diff
@@ -30,11 +30,11 @@ public function __construct(ContentTypeService $contentTypeService, UserService
     protected function configure()
     {
         $this->setDefinition([
-            new InputArgument('identifier', InputArgument::REQUIRED, 'Content Type identifier'),
-            new InputArgument('group_identifier', InputArgument::REQUIRED, 'Content Type group identifier'),
+            new InputArgument('identifier', InputArgument::REQUIRED, 'content type identifier'),
+            new InputArgument('group_identifier', InputArgument::REQUIRED, 'content type group identifier'),
```
Comment (dabrt) [inline code_samples/api/public_php_api/src/Command/CreateContentTypeCommand.php:34]:
```suggestion
            new InputArgument('identifier', InputArgument::REQUIRED, 'Content type identifier'),
            new InputArgument('group_identifier', InputArgument::REQUIRED, 'Content type group identifier'),
```

File: code_samples/api/public_php_api/src/Command/CreateContentTypeCommand.php [hunk 2]
```diff
@@ -51,7 +51,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         try {
             $contentTypeGroup = $this->contentTypeService->loadContentTypeGroupByIdentifier($groupIdentifier);
         } catch (\eZ\Publish\API\Repository\Exceptions\NotFoundException $e) {
-            $output->writeln("Content Type group with identifier $groupIdentifier not found");
+            $output->writeln("content type group with identifier $groupIdentifier not found");
```
Comment (dabrt) [inline code_samples/api/public_php_api/src/Command/CreateContentTypeCommand.php:54]:
```suggestion
            $output->writeln("Content type group with identifier $groupIdentifier not found");
```

File: code_samples/api/public_php_api/src/Command/CreateContentTypeCommand.php [hunk 3]
```diff
@@ -81,7 +81,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $this->contentTypeService->publishContentTypeDraft($contentTypeDraft);
-        $output->writeln("Content type '$contentTypeIdentifier' with ID $contentTypeDraft->id created");
+        $output->writeln("content type '$contentTypeIdentifier' with ID $contentTypeDraft->id created");
```
Comment (dabrt) [inline code_samples/api/public_php_api/src/Command/CreateContentTypeCommand.php:84]:
```suggestion
        $output->writeln("Content type '$contentTypeIdentifier' with ID $contentTypeDraft->id created");
```

File: code_samples/api/public_php_api/src/Command/FindComplexCommand.php [hunk 4]
```diff
@@ -28,10 +28,10 @@ public function __construct(SearchService $searchService, LocationService $locat
     protected function configure()
     {
         $this
-            ->setDescription('Lists content belonging to the provided Content Type.')
+            ->setDescription('Lists content belonging to the provided content type.')
             ->setDefinition([
                 new InputArgument('locationId', InputArgument::REQUIRED, ''),
-                new InputArgument('contentTypeIdentifier', InputArgument::REQUIRED, 'Content Type identifier'),
+                new InputArgument('contentTypeIdentifier', InputArgument::REQUIRED, 'content type identifier'),
```
Comment (dabrt) [inline code_samples/api/public_php_api/src/Command/FindComplexCommand.php:34]:
```suggestion
                new InputArgument('contentTypeIdentifier', InputArgument::REQUIRED, 'Content type identifier'),
```

File: code_samples/back_office/menu/menu_item/templates/themes/admin/all_content_list.html.twig [hunk 5]
```diff
@@ -47,7 +47,7 @@
             headline: 'Content List',
             head_cols: [
                 { content: 'Content name'|trans },
-                { content: 'Content Type'|trans },
+                { content: 'content type'|trans },
```
Comment (dabrt) [inline code_samples/back_office/menu/menu_item/templates/themes/admin/all_content_list.html.twig:50]:
```suggestion
                { content: 'Content type'|trans },
```

File: docs/administration/back_office/back_office_elements/reusable_components.md [hunk 6]
```diff
@@ -263,7 +263,7 @@ See the example:
         { has_checkbox: true },
         { has_icon: true },
         { content: 'article.list.name'|trans|desc('Name') },
-        { content: 'article.list.content_type'|trans|desc('Content Type') },
+        { content: 'article.list.content_type'|trans|desc('content type') },
```
Comment (dabrt) [inline docs/administration/back_office/back_office_elements/reusable_components.md:266]:
```suggestion
        { content: 'article.list.content_type'|trans|desc('Content type') },
```

File: docs/administration/back_office/back_office_menus/back_office_menus.md [hunk 7]
```diff
@@ -53,7 +53,7 @@ You can listen to the following events:
 || `ConfigureMenuEvent::OBJECT_STATE_GROUP_EDIT_SIDEBAR_RIGHT` |
 || `ConfigureMenuEvent::OBJECT_STATE_CREATE_SIDEBAR_RIGHT` |
 || `ConfigureMenuEvent::OBJECT_STATE_EDIT_SIDEBAR_RIGHT` |
-| Content Types | `ConfigureMenuEvent::CONTENT_TYPE_GROUP_CREATE_SIDEBAR_RIGHT` |
+| content types | `ConfigureMenuEvent::CONTENT_TYPE_GROUP_CREATE_SIDEBAR_RIGHT` |
```
Comment (dabrt) [inline docs/administration/back_office/back_office_menus/back_office_menus.md:56]:
```suggestion
| Content types | `ConfigureMenuEvent::CONTENT_TYPE_GROUP_CREATE_SIDEBAR_RIGHT` |
```

**Review by dabrt — APPROVED** [review]

File: docs/api/rest_api/rest_api_reference/input/ez_original.raml [hunk 8]
```diff
@@ -1568,51 +1568,51 @@ version: 2.5
                     example: !include examples/content/typegroups/POST/ContentTypeGroupInput.xml.example
             responses:
                 201:
-                    description: Content Type group created.
+                    description: content type group created.
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez_original.raml:1571]:
```suggestion
                    description: Content type group created.
```
New sentence? I'm not sure 🤷

File: docs/api/rest_api/rest_api_reference/input/ez_original.raml [hunk 9]
```diff
@@ -1624,46 +1624,46 @@ version: 2.5
                         example: !include examples/content/typegroups/content_type_group_id/PATCH/ContentTypeGroupInput.xml.example
                 responses:
                     200:
-                        description: Content Type group updated.
+                        description: content type group updated.
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez_original.raml:1627]:
```suggestion
                        description: Content type group updated.
```

File: docs/api/rest_api/rest_api_reference/input/ez_original.raml [hunk 10]
```diff
@@ -1697,27 +1697,27 @@ version: 2.5
                             example: !include examples/content/typegroups/content_type_group_id/types/POST/ContentTypeCreate.xml.example
                     responses:
                         201:
-                            description: Content Type created.
+                            description: content type created.
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez_original.raml:1700]:
```suggestion
                            description: Content type created.
```

File: docs/api/rest_api/rest_api_reference/input/ez_original.raml [hunk 11]
```diff
@@ -1870,39 +1870,39 @@ version: 2.5
...
+                    displayName: Publish content type draft
+                    description: Publishes a content type draft. PUBLISH or POST with header X-HTTP-Method-Override PUBLISH.
                     responses:
                         200:
                             body:
                                 application/vnd.ibexa.api.ContentType+xml:
                                     type: ContentType
                                     example: !include examples/content/types/content_type_id/draft/PUBLISH/ContentType.xml.example
-                            description: Content Type draft published.
+                            description: content type draft published.
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez_original.raml:1885]:
```suggestion
                            description: Content type draft published.
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 12]
```diff
@@ -65,13 +65,13 @@ Content:
...
-            description: 'Remote ID of the Content Type.'
+            description: 'Remote ID of the content type.'
             type: string
         _id:
-            description: 'Unique ID of the Content Type.'
+            description: 'Unique ID of the content type.'
             type: integer
         ContentType:
-            description: 'Content Type.'
+            description: 'content type.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:74]:
```suggestion
            description: 'Content type.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 13]
```diff
@@ -255,44 +255,44 @@ ContentStruct:
             description: 'Adds a field to the field collection. This method could also be implemented by a magic setter so that $fields[$fieldDefIdentifier][$language] = $value or without language $fields[$fieldDefIdentifier] = $value is an equivalent call.'
             type: string
 ContentType:
-    description: 'This class represents a Content Type.'
+    description: 'This class represents a content type.'
     type: BaseObject
     properties:
         id:
-            description: 'Content Type ID.'
+            description: 'content type ID.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:262]:
```suggestion
            description: 'Content type ID.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 14]
```diff
@@ -395,14 +395,14 @@ ContentTypeGroup:
             description: 'Modifier User ID.'
             type: BaseObject
         ContentTypes:
-            description: 'Content Types.'
+            description: 'content types.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:398]:
```suggestion
            description: 'Content types.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 15]
```diff
@@ -412,32 +412,32 @@ ContentTypeGroupCreateStruct:
...
 ContentTypeGroupList:
-    description: 'List of Content Type groups.'
+    description: 'List of content type groups.'
     type: BaseObject
     properties:
         ContentTypeGroup:
             type: ContentTypeGroup[]
 ContentTypeGroupRef:
-    description: 'Content Type group reference.'
+    description: 'content type group reference.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:421]:
```suggestion
    description: 'Content type group reference.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 16]
```diff
@@ -447,44 +447,44 @@ ContentTypeGroupUpdateStruct:
             description: 'If set this value overrides the current time for modified.'
             type: datetime
 ContentTypeInfo:
-    description: 'This class stores Content Type information.'
+    description: 'This class stores content type information.'
     type: BaseObject
     properties:
         id:
-            description: 'Content Type ID.'
+            description: 'content type ID.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:454]:
```suggestion
            description: 'Content type ID.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 17]
```diff
@@ -1274,7 +1274,7 @@ PasswordValidationContext:
     type: object
     properties:
         contentType:
-            description: 'Content Type of the password owner.'
+            description: 'content type of the password owner.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:1277]:
```suggestion
            description: 'Content type of the password owner.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 18]
```diff
@@ -1476,16 +1476,16 @@ Root:
             description: 'Content by the given remote ID.'
             type: BaseObject
         contentTypes:
-            description: 'Content Types.'
+            description: 'content types.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:1479]:
```suggestion
            description: 'Content types.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 19]
```diff
@@ -1476,16 +1476,16 @@ Root:
...
             type: BaseObject
         contentTypeByIdentifier:
-            description: 'Content Type by the given identifier.'
+            description: 'content type by the given identifier.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:1482]:
```suggestion
            description: 'Content type by the given identifier.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 20]
```diff
@@ -1476,16 +1476,16 @@ Root:
...
             type: BaseObject
         contentTypeGroups:
-            description: 'Content Type Groups.'
+            description: 'content type Groups.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:1485]:
```suggestion
            description: 'Content type Groups.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 21]
```diff
@@ -1476,16 +1476,16 @@ Root:
...
             type: BaseObject
         contentTypeGroupByIdentifier:
-            description: 'Content Type Groups by the given identifier.'
+            description: 'content type Groups by the given identifier.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:1488]:
```suggestion
            description: 'Content type Groups by the given identifier.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 22]
```diff
@@ -2039,7 +2039,7 @@ Value:
             description: 'Language code.'
             type: string
         '#text':
-            description: 'Content Type description.'
+            description: 'content type description.'
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:2042]:
```suggestion
            description: 'Content type description.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-content-types.raml [hunk 23]
```diff
@@ -210,47 +210,47 @@ get:
...
+            displayName: Publish content type draft
+            description: Publishes a content type draft. PUBLISH or POST with header X-HTTP-Method-Override PUBLISH.
             responses:
                 200:
                     body:
                         application/vnd.ibexa.api.ContentType+xml:
                             type: ContentType
                             example: !include examples/content/types/content_type_id/draft/PUBLISH/ContentType.xml.example
-                    description: Content Type draft published.
+                    description: content type draft published.
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-content-types.raml:225]:
```suggestion
                    description: Content type draft published.
```

File: docs/api/rest_api/rest_api_reference/input/ez-content-typegroups.raml [hunk 24]
```diff
@@ -51,7 +51,7 @@ post:
             example: !include examples/content/typegroups/POST/ContentTypeGroupInput.json.example
     responses:
         201:
-            description: Content Type group created.
+            description: content type group created.
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-content-typegroups.raml:54]:
```suggestion
            description: Content type group created.
```

File: docs/api/rest_api/rest_api_reference/input/ez-content-typegroups.raml [hunk 25]
```diff
@@ -116,7 +116,7 @@ post:
                 example: !include examples/content/typegroups/content_type_group_id/PATCH/ContentTypeGroupInput.json.example
         responses:
             200:
-                description: Content Type group updated.
+                description: content type group updated.
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-content-typegroups.raml:119]:
```suggestion
                description: Content type group updated.
```

File: docs/api/rest_api/rest_api_reference/input/ez-content-typegroups.raml [hunk 26]
```diff
@@ -195,14 +195,14 @@ post:
                     example: !include examples/content/typegroups/content_type_group_id/types/POST/ContentTypeCreate.json.example
             responses:
                 201:
-                    description: Content Type created.
+                    description: content type created.
```
Comment (mnocon) [inline docs/api/rest_api/rest_api_reference/input/ez-content-typegroups.raml:198]:
```suggestion
                    description: Content type created.
```

**Review by mnocon — COMMENTED** [review]
Huge PR, thank you for taking care of this 😄 

Question about the autogenerated REST reference - were these changes done manually or were they introduced by the tool? Can we regenerate the reference to make sure we don't lose these changes?

**Review by mnocon — APPROVED** [review]

**Pull request merged**